tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2203,6 +2186,126 @@ public void handlePath(String xpath) {
+                boolean revertToBaseline = (steps >= 2) && wantRevertToBaseline(xpath, vxmlValue);
+                if (!revertToBaseline) {
+                    String fullXPath = cldrFileToFilter.getFullXPath(xpath);
+                    replace(fullXPath, fullXPath, vxmlValue);
+                }
+            }
+
+            /**
+             * 	Revert each L2+ locale/path VXML value of ↑↑↑ to the baseline value, where all the following criteria are met:
+             * 	1	The baseline value is a hard value (can use unresolved file for this)
+             * 	2	The survey tool did not allow changes in the locale/path in v43
+             * 	    ▪ That is, other than allowed locales with Person Names, Turkey, missing, and errors
+             *    	▪ NOTE: we should be able to check missing/errors by running the CheckCLDR tests on the baseline data. Most locales only had PN & Turkey, but some had errors or missing values.
+             * 	3	The bailey value is fetched from a different path AND from an ancestor locale.
+             * 	    ▪ Ideally we would look at intervening paths also, but I think this is sufficient.
+             */
+            private boolean wantRevertToBaseline(String xpath, String vxmlValue) {
+                if (!CldrUtility.INHERITANCE_MARKER.equals(vxmlValue)) {
+                    // criterion zero: if vxml value is not ↑↑↑, don't revert to baseline
+                    return false;
+                }
+                String baselineValue = cldrFileToFilter.getStringValue(xpath);
+                if (baselineValue == null || CldrUtility.INHERITANCE_MARKER.equals(baselineValue)) {
+                    // criterion 1: if baseline value is not a hard value, don't revert to baseline
+                    return false;
+                }
+                String localeID = cldrFileToFilter.getLocaleID();
+                if (changesWereAllowed(localeID, xpath)) {
+                    // criterion 2: if Survey Tool did allow changes in the locale/path in v43, don't revert to baseline
+                    return false;
+                }
+                Output<String> inheritancePathWhereFound = new Output<>();
+                Output<String> localeWhereFound = new Output<>();
+                cldrFileToFilter.getBaileyValue(xpath, inheritancePathWhereFound, localeWhereFound);
+                if (localeID.equals(inheritancePathWhereFound.value) || xpath.equals(inheritancePathWhereFound.value)) {
+                    // criterion 3: if bailey value is not from different path and locale, don't revert to baseline
+                    return false;
+                }
+                return true;
+            }
+
+            private boolean changesWereAllowed(String localeID, String xpath) {
+                boolean isError = pathHasError(localeID, xpath);
+                String oldValue = baselineFileUnresolved.getWinningValue(xpath);
+                boolean isMissing = (oldValue == null);
```

Comment:
if old value = null or if path.contains @draft=

Reply:
needs to be the full-vxml-path (extra arg)

Reply:
could pass in the draft status of the full-vxml-xpath as arg to this method

Reply:
`CLDRFile.DraftStatus.forXPath(fullXpath) == CLDRFile.DraftStatus.provisional`

Reply:
@btangmu you might want to add this to CLDRFile.DraftStatus:

```java
/**
 * provisional.orWorse(x) - true for unconfirmed, provisional…  false for contributed, approved
 */
public boolean orWorse(DraftStatus d) {
    return d.ordinal() <= this.ordinal();
}
```

Reply:
then you could do:

`if ( CLDRFile.DraftStatus.provisional.orWorse(CLDRFile.DraftStatus.forXPath(fullXPath)))`

true if provisional or worse

Reply:
I'd rather avoid changes outside of CLDRModify until the dust has settled, so for the next PR I'll do something equivalent without changing CLDRFile:

`boolean isMissing = (oldValue == null || CLDRFile.DraftStatus.forXpath(fullXPath).ordinal() <= CLDRFile.DraftStatus.provisional.ordinal());`

Let's remember the idea for orWorse().

Reply:
That's fine for now.

BTW, later, better would be 

```
        public boolean isWorseThan(DraftStatus target) {
            return compareTo(target) < 0;
        }

```
Usage would be 

`DraftStatus.forXpath(fullXPath).isWorseThan(DraftStatus.contributed)`